packages/evolution-common/src/services/widgets/WidgetConfig.ts

```diff
@@ -282,6 +288,48 @@ export type TextWidgetConfig = {
     conditional?: ParsingFunction<boolean | [boolean] | [boolean, unknown]>;
 };
 
+export type ButtonWidgetConfig = {
+    type: 'button';
+    // FIXME What is this path used for? Document and/or type further
```

Comment:
Change this for: // Note: The 'path' is required to resolve ${relativePath} in conditional expressions.

Reply:
fait